src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -103,125 +106,132 @@ namespace AirflowNetwork {
-        DMP = 10, // Distribution system damper component
-        ELR = 11, // Distribution system effective leakage ratio component
-        CPD = 12, // Distribution system constant pressure drop component
-        COI = 13, // Distribution system coil component
-        TMU = 14, // Distribution system terminal unit component
-        EXF = 15, // Zone exhaust fan
-        HEX = 16, // Distribution system heat exchanger
-        HOP = 17, // Horizontal opening component
-        RVD = 18, // Reheat VAV terminal damper
-        OAF = 19, // Distribution system OA
-        REL = 20  // Distribution system relief air
+        Unassigned = -1,
+        DOP, // Detailed large opening component
+        SOP, // Simple opening component
+        SCR, // Surface crack component
+        SEL, // Surface effective leakage ratio component
+        PLR, // Distribution system crack component
+        DWC, // Distribution system duct component
+        CVF, // Distribution system constant volume fan component
+        FAN, // Distribution system detailed fan component
+        MRR, // Distribution system multiple curve fit power law resistant flow component
+        DMP, // Distribution system damper component
+        ELR, // Distribution system effective leakage ratio component
+        CPD, // Distribution system constant pressure drop component
+        COI, // Distribution system coil component
+        TMU, // Distribution system terminal unit component
+        EXF, // Zone exhaust fan
+        HEX, // Distribution system heat exchanger
+        HOP, // Horizontal opening component
+        RVD, // Reheat VAV terminal damper
+        OAF, // Distribution system OA
+        REL  // Distribution system relief air
     };
 
     enum class ComponentType
     {
-        DOP = 1, // Detailed large opening component
-        SOP,     // Simple opening component
-        SCR,     // Surface crack component
-        SEL,     // Surface effective leakage ratio component
-        PLR,     // Distribution system crack component
-        DWC,     // Distribution system duct component
-        CVF,     // Distribution system constant volume fan component
-        FAN,     // Distribution system detailed fan component
-        MRR,     // Distribution system multiple curve fit power law resistant flow component
-        DMP,     // Distribution system damper component
-        ELR,     // Distribution system effective leakage ratio component
-        CPD,     // Distribution system constant pressure drop component
-        COI,     // Distribution system coil component
-        TMU,     // Distribution system terminal unit component
-        EXF,     // Zone exhaust fan
-        HEX,     // Distribution system heat exchanger
-        HOP,     // Horizontal opening component
-        RVD,     // Reheat VAV terminal damper
-        OAF,     // Distribution system OA
-        REL      // Distribution system relief air
+        Unassigned = -1,
```

Comment:
Revert these changes, there is no need for Unassigned and the new design removes it as a possibility.